### DISCUSSION TITLE
linking service account to conocurse team when deploying concourse to EKS on AWS.

### DISCUSSION THREAD

**Discussion by bonzofenix** [concourse/concourse#6825] [Help & Support]
Is there a way to give IAM permissions and linking service account to a team or a concourse cluster with the current implementation of the concourse-chart? I open an issue related to this in the concourse project:

https://github.com/concourse/concourse-chart/issues/239

**Comment by bonzofenix** [comment]
After doing some extra research I realise that in order to use the IAM role from a pod, I would need to pass some worker env variables to a task. Is there a way for a privilaged/unprivilaged container to access the host(worker) envs?

**Reply by taylorsilva** [reply]
Nope, no way to pass those along afaik
